www/layouts/_default/single.html

```diff
@@ -14,4 +14,176 @@
     </p>
   {{- end -}}
 
+  {{ with .Params.comments }}
+  <section id="comments" class="article-content">
+    <h2>Commentaires</h2>
+    <p>Avec un compte sur le Fediverse ou Mastodon, vous pouvez répondre à ce <a href="https://{{ .host }}/@{{ .username }}/{{ .id }}">post</a>. Mastodon étant décentralisé, vous pouvez utiliser votre compte existant hébergé sur un autre serveur Mastodon ou une plateforme compatible si vous n'avez pas de compte sur celui-ci.</p>
+  
+    <p id="mastodon-comments-list"><button id="load-comment">Charger les commentaires</button></p>
+    <div id="comments-wrapper">
+      <noscript><p>Charger les commentaires nécessite JavaScript. Essayez d'activer Javascript et rechargez la page, ou visitez <a href="https://{{ .host }}/@{{ .username }}/{{ .id }}">le post original</a> sur Mastodon.</p></noscript>
+    </div>
+    <noscript>Vous devez activer Javascript pour voir les commentaires.</noscript>
+    <script src="/js/purify.min.js"></script>
+    <script type="text/javascript">
```

Comment:
would you mind putting this script in a separate file, e.g. `comments.js` or `mastodon-comments.js` please ?